src/azul/service/manifest_service.py

```diff
@@ -1388,13 +1389,28 @@ def _all_docs_sorted(self) -> Iterable[JSON]:
         request = request.params(preserve_order=True).sort('entity_id.keyword')
         for hit in request.scan():
             doc = self._hit_to_doc(hit)
+            self._set_drs_uri(doc)
             yield doc
             file_ = one(doc['contents']['files'])
             for related in file_['related_files']:
                 related_doc = copy_json(doc)
                 related_doc['contents']['files'] = [{**file_, **related}]
                 yield related_doc
 
+    def _set_drs_uri(self, doc: AnyJSON) -> None:
```

Comment:
This would silently do nothing if we renamed the field and forgot to update this code. The other manifests have a similar problem in this area but we can do better starting now.

I think the code should go into the PFB module. You can parameterize the converter with a plugin reference.

The field extraction should break if the field doesn't exist.

The field should be named `drs_uri` not `drs_path` in the output PFB.

Reply:
Can you also add FIXME for the fact that the other manifests still calls this drs_path, not drs_uri?